release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -0,0 +1,832 @@
+|GA
+|GA
+|GA
+
+|StatefulSets
+|GA
+|GA
+|GA
+
+|xref:../admin_guide/quota.adoc#limited-resources-quota[Explicit Quota]
+|GA
+|GA
+|GA
+
+|xref:../architecture/additional_concepts/storage.adoc#pv-mount-options[Mount Options]
+|
+|GA
+|GA
+
+|System Containers for docker, CRI-O
+|Dropped
+|-
+|-
+
+|xref:../install/running_install.adoc#running-the-advanced-installation-system-container[System Container for installer and Kubelet]
+|GA
+|Dropped
+|-
+
+|Hawkular Agent
+|-
+|-
+|-
+
+|Pod PreSets
+|-
+|-
+|-
+
+|xref:../admin_guide/overcommit.adoc#configuring-reserve-resources[experimental-qos-reserved]
+|TP
+|TP
+|TP
+
+|xref:../admin_guide/sysctls.adoc#admin-guide-sysctls[Pod sysctls]
+|TP
+|TP
+|TP
+
+|xref:../install_config/master_node_configuration.adoc#master-node-config-audit-config[Central Audit]
+|GA
+|GA
+|GA
+
+|xref:../admin_guide/managing_networking.adoc#enabling-static-ips-for-external-project-traffic[Static IPs for External Project Traffic]
+|GA
+|GA
+|GA
+
+|xref:../dev_guide/templates.adoc#waiting-for-template-readiness[Template Completion Detection]
+|GA
+|GA
+|GA
+
+|xref:../cli_reference/basic_cli_operations.adoc#object-types[`replicaSet`]
+|GA
+|GA
+|GA
+
+|xref:../install_config/aggregate_logging.adoc#aggregated-fluentd[Mux]
+|TP
+|TP
+|TP
+
+|Clustered MongoDB Template
+|-
+|-
+|-
+
+|Clustered MySQL Template
+|-
+|-
+|-
+
+|xref:../dev_guide/managing_images.adoc#using-is-with-k8s[Image Streams with Kubernetes Resources]
+|GA
+|GA
+|GA
+
+|xref:../dev_guide/device_manager.adoc#using-device-manager[Device Manager]
+|TP
+|GA
+|GA
+
+|xref:ocp-311-pv-resize[Persistent Volume Resize]
+|TP
+|TP
+|TP
+
+|xref:../scaling_performance/managing_hugepages.adoc#scaling-performance-managing-huge-pages[Huge Pages]
+|TP
+|GA
+|GA
+
+|xref:../scaling_performance/using_cpu_manager.adoc#scaling-performance-using-cpu-manager[CPU Manager]
+|TP
+|GA
+|GA
+
+|xref:../dev_guide/device_plugins.adoc#using-device-plugins[Device Plug-ins]
+|TP
+|GA
+|GA
+
+|xref:ocp-311-syslog-output-plugin-for-fluentd[syslog Output Plug-in for fluentd]
+|TP
+|TP
+|TP
+
+|xref:ocp-311-container-storage-Interface[Container Storage Interface (CSI)]
+|-
+|TP
+|TP
+
+|xref:ocp-311-pv-provisioning-using-openstack-manilla[Persistent Volume (PV) Provisioning Using OpenStack Manila]
+|-
+|TP
+|TP
+
+|xref:ocp-311-node-problem-detector[Node Problem Detector]
+|-
+|TP
+|TP
+
+|xref:ocp-311-local-ephemeral-storage[Protection of Local Ephemeral Storage]
+|-
+|TP
+|TP
+
+|xref:ocp-311-the-descheduler[Descheduler]
+|-
+|TP
+|TP
+
+|xref:ocp-311-podman[Podman]
+|-
+|TP
+|TP
+
+|xref:ocp-311-kuryr[Kuryr CNI Plug-in]
+|-
+|TP
+|TP
+
+|xref:ocp-311-control-sharing-pid-namespace-between-containers[Sharing Control of the PID Namespace]
+|-
+|TP
+|TP
+
+|Cluster Administrator console
+|-
+|-
+|TP
+
+|Cluster Autoscaling (AWS Only)
+|-
+|-
+|GA
+
+|Operator Lifecycle Manager
+|-
+|-
+|TP
+
+|Etcd Operator for Tenants
+|-
+|-
+|TP
+
+|Prometheus Operator for Tenants
+|-
+|-
+|TP
+
```

Comment:
I think the `Operator-SDK` feature is TP for 3.11, but I could not find it. Document feature card: https://trello.com/c/rU4yP8Cu/1229-formal-documentation-for-operator-sdk.